pkgs/by-name/bs/git-blame-someone-else/package.nix

```diff
@@ -0,0 +1,48 @@
+{ callPackage
+, lib
+, stdenv
+, fetchurl
+, nixos
+, testers
+, versionCheckHook
+, git-blame-someone-else
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "git-blame-someone-else";
+
+  src = fetchurl {
+    url = "https://github.com/jayphelps/git-blame-someone-else/archive/refs/heads/master.zip";
+    hash = "sha256-jZkUKv2SV28wsM18tCqNxoCZmLxdYH2Idh9RLibH2yA=";
+  };
+
+  doCheck = true;
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  # Give git-blame-someone-else some install checks for testing purpose.
+  postInstallCheck = ''
+    stat "''${!outputBin}/bin/${finalAttrs.meta.mainProgram}"
```

Comment:
im not sure that this is a useful test

Reply:
Sorry, was using gnu hello as a template